pydis_site/apps/staff/tests/test_deletedmessage_filters.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase
+from django.utils import timezone
+
+from ..templatetags import deletedmessage_filters
+
+
+class DeletedMessageFilterTests(TestCase):
+    def test_hex_colour_filter(self):
+        self.assertEqual(deletedmessage_filters.hex_colour(0), "#000000")
+        self.assertEqual(deletedmessage_filters.hex_colour(255), "#0000FF")
+        self.assertEqual(deletedmessage_filters.hex_colour(65280), "#00FF00")
+        self.assertEqual(deletedmessage_filters.hex_colour(16711680), "#FF0000")
+        self.assertEqual(deletedmessage_filters.hex_colour(16777215), "#FFFFFF")
```

Comment:
maybe some constants would help this look a bit less insane.

Reply:
I've changed it to use an IntEnum of the colors.